packages/apps/human-app/frontend/src/modules/worker/email-verification/components/email-verification-process.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  PageCardError,
+  PageCardLoader,
+} from '@/shared/components/ui/page-card';
+import { getErrorMessageForError } from '@/shared/errors';
+import { useVerifyEmailQuery } from '../hooks';
+import { EmailVerificationSuccessMessage } from './email-verification-success-message';
+
+interface EmailVerificationProcessProps {
+  token: string;
+}
+
+function useEmailVerification(token: string) {
```

Comment:
IMO you don't need to go that granular and this code can be part of `EmailVerificationProcess`